react/PopupOpener/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// customized function to center a popup window
+// source https://stackoverflow.com/a/16861050
+export function popupCenter(url, title, w, h) {
+  /* global screen */
+  // Fixes dual-screen position
+  //                      Most browsers      Firefox
+  var dualScreenLeft = window.screenLeft || screen.left
+  var dualScreenTop = window.screenTop || screen.top
+
+  var width = window.innerWidth
+    ? window.innerWidth
+    : document.documentElement.clientWidth
+      ? document.documentElement.clientWidth
+      : screen.width
```

Comment:
Not blocking : `window.innerWidth || document.documentElement.clientWidth || screen.width`